### DISCUSSION TITLE
Add apps to upper bar on Android Auto and screen brightness

### DISCUSSION THREAD

**Discussion by SOFO888** [AndreyPavlenko/Fermata#383] [Ideas]
Dear Andrey, 

Thank you for Fermata. 
I installed version 1.9.8 today and was happy to see the possibility to mirror my phone screen. 
I used already Screen2Auto. 
But Fermata Mirroring works way better. 
It is faster and the touch function is almost perfect. Moreover it is a Navigation app, which means I can use it as start app. Thank you. 

I miss some features that I had on Screen2Auto: 
- apps on upper bar: on Screen2Auto there was an app bar above the mirrored screen. I saw that you integrated Fermata player. Is it possible to add the possibility to add more apps (user chosen apps)? Is it possible that this bar is always visible? Perhaps with the possibility to adjust the logo size? 
- screen brightness: is it possible to make the auto screen brightness during mirroring function optional? 

Thank you very much. 

Best regards

**Comment by AndreyPavlenko** [comment]
In the FS Mirror app I don't have full control over the toolbar, but adding more buttons is possible.
I'll add settings for brightness and rotation in further releases.

**Comment by SOFO888** [comment]
Thank you very much.

**Comment by SOFO888** [comment]
Hello Andrey, 

I tried around with FS mirror the past 2 days and it is really great. 
Is it also possible to change the starting app (currently Fermata Player) in a future release? Like having the home screen. 

Thanks a lot.

**Reply by AndreyPavlenko** [reply]
This option to be added in the next release.

**Comment by AndreyPavlenko** [comment]
This pre-release version can temporary restore the brightness on the phone screen touch - https://app.box.com/s/ns86n7vc3ssaltli0mmibrydnyvbr6bs
